examples/mnist-rs/host/src/commands/infer.rs

```diff
@@ -0,0 +1,80 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use clap::Parser;
+use image::EncodableLayout;
+use optee_teec::Context;
+use proto::{Image, IMAGE_SIZE};
+
+#[derive(Parser, Debug)]
+pub struct Args {
+    /// The path of the model.
+    #[arg(short, long)]
+    model: String,
+    /// The path of the input binary, must be 768 byte binary, can be multiple
+    #[arg(short, long)]
+    binary: Vec<String>,
+    /// The path of the input image, must be dimension of 28x28, can be multiple
+    #[arg(short, long)]
+    image: Vec<String>,
+}
+
+pub fn execute(args: &Args) {
+    let model_path = std::path::absolute(&args.model).unwrap();
+    println!("Load model from \"{}\"", model_path.display());
+    let record = std::fs::read(&model_path).unwrap();
+    let mut ctx = Context::new().unwrap();
+    let mut caller = crate::tee::Model::new(&mut ctx, &record).unwrap();
```

Comment:
Could we handle the error instead of `unwrap()` to trigger panic?
There're some `unwrap()` in other files too.